brokerapi/brokers/api_service/definition.go

```diff
@@ -35,35 +35,27 @@ func ServiceDefinition() *broker.ServiceDefinition {
 	}
 
 	return &broker.ServiceDefinition{
-		Name: models.MlName,
-		DefaultServiceDefinition: `
-		{
-      "id": "5ad2dce0-51f7-4ede-8b46-293d6df1e8d4",
-      "description": "Machine Learning APIs including Vision, Translate, Speech, and Natural Language.",
-      "name": "google-ml-apis",
-      "bindable": true,
-      "plan_updateable": false,
-      "metadata": {
-        "displayName": "Google Machine Learning APIs",
-        "longDescription": "Machine Learning APIs including Vision, Translate, Speech, and Natural Language.",
-        "documentationUrl": "https://cloud.google.com/ml/",
-        "supportUrl": "https://cloud.google.com/support/",
-        "imageUrl": "https://cloud.google.com/_static/images/cloud/products/logos/svg/machine-learning.svg"
-      },
-      "tags": ["gcp", "ml"],
-      "plans":  [
-        {
-         "id": "be7954e1-ecfb-4936-a0b6-db35e6424c7a",
-         "service_id": "5ad2dce0-51f7-4ede-8b46-293d6df1e8d4",
```

Comment:
Talked with @josephlewis42 , `service_id` and `display_name` were never used in the first place